tests/Integration/Behaviour/Features/Scenario/Category/category_management.feature

```diff
@@ -594,6 +604,7 @@ Feature: Category Management
       | meta title[fr-FR]             | meta title french              |
       | meta keywords[en-US]          | meta,keyword,english           |
       | meta keywords[fr-FR]          | meta,keyword,french            |
+      | redirect type                 | 404                            |
```

Comment:
When switching to 404 (and 410) the target ID should be reset to 0 and the behat tests should assert this reset

Reply:
OK perfect then